projects/pgai/pgai/cli.py

```diff
@@ -245,7 +245,6 @@ def install(db_url: str, strict: bool) -> None:
     import pgai
 
     pgai.install(db_url, strict=strict)
-    log.info(f"pgai {__version__} installed")
```

Comment:
Did you mean to remove this given it was only added today in https://github.com/timescale/pgai/commit/a8f5f8d66d25ee93427617e60d06383101593d5b?

Reply:
nope. good catch. thanks

Reply:
stupid merge conflicts